ui/shared/components/panel/family/FamilyReads.jsx

```diff
@@ -167,6 +167,236 @@ const REFERENCE_LOOKUP = ['37', '38'].reduce((acc, genome) => ({
+    text: 'Norm. GTEx Brain: Cortex',
+  },
+  {
+    data: [
+      {
+        type: COVERAGE_TYPE,
+        url: 'gs://tgg-viewer/ref/GRCh38/gtex_v8/GTEX_frontal_cortex.209_samples.bigWig',
+      },
+      {
+        type: JUNCTION_TYPE,
+        url: 'gs://tgg-viewer/ref/GRCh38/gtex_v8/GTEX_frontal_cortex.209_samples.normalized.junctions.bed.gz',
+      },
+    ],
+    description: 'Highly expressed junctions from all 209 GTEx v3 brain frontal cortex samples.\n The junction-spanning read counts are normalized to represent the average spanning read count per-sample (see formula below).\n Only junctions with rounded normalized spanning read count > 0 are included in this track.\n \n average_unique_reads_per_cortex_sample = (total_unqiue_reads_in_all_cortex_samples / number_of_cortex_samples)\n per_sample_normalized_read_count = raw_read_count * average_unique_reads_per_cortex_sample / total_unqiue_reads_in_this_sample\n normalized read count for junction = sum(per_sample_normalized_read_counts) / number_of_cortex_samples',
+    text: 'Norm. GTEx Brain: Front. Cortex',
+  },
+]
+
+const MAPPABILITY_TRACKS = [
+  {
+    type: COVERAGE_TYPE,
+    url: 'gs://tgg-viewer/ref/GRCh38/mappability/GRCh38_no_alt_analysis_set_GCA_000001405.15-k36_m2.bw',
+    text: '36-mer mappability',
+    description: 'Mappability of 36-mers allowing for 2 mismatches. Generated using the same pipeline as the UCSC hg19 mappability tracks.',
+  },
+  {
+    type: COVERAGE_TYPE,
+    url: 'gs://tgg-viewer/ref/GRCh38/mappability/GRCh38_no_alt_analysis_set_GCA_000001405.15-k50_m2.bw',
+    text: '50-mer mappability',
+    description: 'Mappability of 50-mers allowing for 2 mismatches. Generated using the same pipeline as the UCSC hg19 mappability tracks.',
+  },
+  {
+    type: COVERAGE_TYPE,
+    url: 'gs://tgg-viewer/ref/GRCh38/mappability/GRCh38_no_alt_analysis_set_GCA_000001405.15-k75_m2.bw',
+    text: '75-mer mappability',
+    description: 'Mappability of 75-mers allowing for 2 mismatches. Generated using the same pipeline as the UCSC hg19 mappability tracks.',
+  },
+  {
+    type: COVERAGE_TYPE,
+    url: 'gs://tgg-viewer/ref/GRCh38/mappability/GRCh38_no_alt_analysis_set_GCA_000001405.15-k100_m2.bw',
+    text: '100-mer mappability',
+    description: 'Mappability of 100-mers allowing for 2 mismatches. Generated using the same pipeline as the UCSC hg19 mappability tracks.',
+  },
+  {
+    type: 'annotation',
+    options: {
+      format: 'gtf',
+      height: 100,
+    },
+    url: 'gs://tgg-viewer/ref/GRCh38/segdups/segdups.gtf.gz',
+    text: 'SegDups >1000 bases',
+    description: 'Duplications of >1000 Bases of Non-RepeatMasked Sequence downloaded from UCSC',
+  },
+]
+
+const GTEX_TRACK_OPTIONS = GTEX_TRACKS.map(({ text, description }) => ({ value: text, text, description }))
+
+const MAPPABILITY_TRACK_OPTIONS = MAPPABILITY_TRACKS.map(({ text, description }) => ({ value: text, text, description }))
```

Comment:
I really don't think you need these. The input class itself puts all the options through a mapping to sanitize them so you don't need to remove extra keys here, and it will default the text to the value if none is provided. So if you just change the `text` keys to `value` for you track constants they can be passed directly in as options without this processing

https://github.com/broadinstitute/seqr/blob/master/ui/shared/components/form/Inputs.jsx#L79

Reply:
Sounds good. I didn't know the `value` can be used for the `text`.